stdlib/public/core/DropWhile.swift

```diff
@@ -28,9 +28,11 @@ public struct LazyDropWhileSequence<Base: Sequence> {
     self._base = _base
     self._predicate = predicate
   }
-
 }
 
+@available(*, unavailable)
+extension LazyDropWhileSequence: Sendable {}
```

Comment:
Why isn't this type conditionally Sendable?

Reply:
The predicate it stores is not known to be sendable. Same with `LazyFilter`

Reply:
Ah that's interesting. We had the same problem with some of async variants and they all require the closure to be Sendable but I agree that doesn't make sense here!